integration/integration_test.go

```diff
@@ -39,32 +38,41 @@ func TestRunWithKind(t *testing.T) {
 		ExpectError   bool
 	}{
 		{
-			TestName:      "job",
-			KindCfg:       "./testdata/add-tls-kind-k8s114.yaml",
+			TestName:      "kube-bench",
 			KubebenchYAML: "../job.yaml",
 			ExpectedFile:  "./testdata/job.data",
 		},
 		{
-			TestName:      "job-node",
-			KindCfg:       "./testdata/add-tls-kind-k8s114.yaml",
+			TestName:      "kube-bench-node",
 			KubebenchYAML: "../job-node.yaml",
 			ExpectedFile:  "./testdata/job-node.data",
 		},
 		{
-			TestName:      "job-master",
-			KindCfg:       "./testdata/add-tls-kind-k8s114.yaml",
+			TestName:      "kube-bench-master",
 			KubebenchYAML: "../job-master.yaml",
 			ExpectedFile:  "./testdata/job-master.data",
 		},
 	}
+	ctx, err := setupCluster("kube-bench", "./testdata/add-tls-kind-k8s114.yaml", *timeout)
+	if err != nil {
+		t.Fatalf("failed to setup KIND cluster error: %v", err)
+	}
+	defer func() {
+		ctx.Delete()
+	}()
+
+	if err := loadImageFromDocker(*kubebenchImg, ctx); err != nil {
+		t.Fatalf("failed to load kube-bench image from Docker to KIND error: %v", err)
+	}
+
 	for _, c := range cases {
-		t.Run(c.TestName, func(t *testing.T) {
-			data, err := runWithKind(c.TestName, c.KindCfg, c.KubebenchYAML, *kubebenchImg, timeout, ticker)
+		t.Run(c.TestName, func(tc *testing.T) {
```

Comment:
I think it's more conventional to continue to call it `t` rather than `tc` inside the test function (as you did before)

Reply:
I'm using `tc` (as in test case) so that it's clear I'm not using the outer `t`.

Reply:
The convention is to use `tc` for the test case (see https://blog.golang.org/subtests for an example), and to use `t` for the testing structure. These conventions make code easier to read and re-use, so let's stick with them

Reply:
sounds good.